### PR TITLE
First draft of an article about DKIM

### PR DESCRIPTION
**Pull request by aeden**

### CODE REVIEW

File: content/articles/dkim-record.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+# DKIM Records
+
+* TOC
+{:toc}
+
+---
+
+DKIM stands for DomainKeys Identified Email and provides a way to validate that an organization delivering an email has the right to do so.
```
Comment (jcaudle) [inline content/articles/dkim-record.markdown]: Should there be a space in `DomainKeys`?

File: content/articles/dkim-record.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+This should be inserted into a TXT record. You can do this by following the instructions for [creating a record](http://support.dnsimple.com/articles/advanced-editor/#create-a-record), selecting TXT as the record type, and entering the string you were given in the Content field.
+
+Your provider will also have given you a specific subdomain to use, usually something like this:
+
+   something.\_domainkey
+
+You will enter this subdomain in the "Name" field.
+
+Note that if your provider gives you a fully-qualified name that ends with your domain name you should NOT include your domain name in the "Name" field when you add the TXT record. For example, if you are given `pm.\_domainkey.yourdomain.com` then you will only enter `pm.\_domainkey` in the "Name" field.
```
Comment (jcaudle) [inline content/articles/dkim-record.markdown]: Escape characters aren't needed inside the backticks here. Yay Markdown! :stuck_out_tongue:

**Comment by jcaudle** [comment]
It will definitely be nice to point people at this when we get questions on DKIM. :clap:

File: content/articles/dkim-record.markdown (hunk 1 above)
Comment (aeden) [inline content/articles/dkim-record.markdown]: Nope, it's actually written that way in the RFCs:

- https://www.ietf.org/rfc/rfc4871.txt
- https://tools.ietf.org/html/rfc6376

File: content/articles/dkim-record.markdown (hunk 2 above)
Comment (aeden) [inline content/articles/dkim-record.markdown]: Thanks. I was trying to track down a missing backslash - good to know they are not needed in single quotes.

**Pull request merged**